sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRestClient.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal partial class TextAnalyticsRestClient
+    {
+        internal HttpMessage CreateHealthStatusNextPageRequest(string nextLink, bool? showStats)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendRaw("/text/analytics/v3.1-preview.3", false);
```

Comment:
We will be updating this every time we move to next version? Is this ok to hardcode api version?

Reply:
hmm yeah we will forget... I will update to using https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs for now.

For next milestone I will look into how to modify the swagger in a way that we get pageable for free, so hopefully this will go away.
Issue https://github.com/Azure/azure-sdk-for-net/issues/18373